x11-misc/x11vnc/files/x11vnc.init.d-r1

```diff
@@ -29,8 +29,13 @@ checkconfig() {
 	fi
 	checkpath -q -f -m 0600 -o root:root "${X11VNC_RFBAUTH}"
 
+	# Attempt to find SDDM
+	if [ -d /var/run/sddm/ ]; then
+		# If SDDM is found
+		X11VNC_AUTH_OPTS="-auth /var/run/sddm/*"
+		X11VNC_OPTS="-xkb -noxrecord"
```

Comment:
Overriding `X11VNC_OPTS` completely is a bad idea. Could you explain why these options are required here?

Reply:
You're right. Now these parameters are applied by default and will be superfluous here. Thanks for the good note. Remove the bad line.